tests/management/commands/test_raise_test_exception.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from django.core.management import call_command
+
+
+def test_raise_test_exception():
+    exception_raised = False
+    try:
+        call_command('raise_test_exception')
+    except Exception:
```

Comment:
Please consider using `with self.assertRaises()` code in your work. :)

Reply:
@yeojin-dev notice there isn’t a `self` argument here - I didn’t use it to follow the structure of other tests in the repo. If it makes sense to use django’s `TestCase` here I’d be happy to do it, I just wasn’t sure why it wasn’t used elsewhere.

Reply:
Or maybe:

```python
with pytest.raises(SomeException):
    call_command('raise_test_exception')
```

Reply:
@trbs great suggestion - I just updated the code and created a `DjangoExtensionsTestException` Exception subclass.